dev-tools/mage/common.go

```diff
@@ -353,6 +353,71 @@ func unzip(sourceFile, destinationDir string) error {
 	return nil
 }
 
+// Tar compress a directory using tar + gzip algorithms
+func Tar(src string, targetFile string) error {
+	fmt.Printf(">> creating TAR file from directory: %s, target: %s\n", src, targetFile)
+	var buf bytes.Buffer
```

Comment:
How big do this files get? It might be better to do a `os.Create` and write directing to the file instead of buffering in memory.

Reply:
I'd say they are not big. My observation says that they could be in the order of tens of Mb?

Reply:
Done in 41300be